services/vehicle/src/main/java/com/wedriveu/services/vehicle/entity/Vehicle.java

```diff
@@ -5,19 +5,28 @@
 
 import java.util.Date;
 
+import static com.wedriveu.services.shared.utilities.Constants.*;
+
 /**
- * Created by Michele on 12/07/2017.
+ * @author Marco Baldassarri
+ * @author Michele
+ * @since 02/08/2017
  */
 public class Vehicle {
```

Comment:
Since you modified the `Vehicle` i can tell you that maybe we need to define all `Vehicle` status as constants, so please see `Booking` class, or define a special enumerator.